### ISSUE TITLE
Investigate definition of pimienta coefficients

### ISSUE THREAD

**Issue by amacki3** [poliastro/poliastro#1472]
In the function pimienta_coe, which has been implemented according to the paper "Accurate Kepler equation solver without transcendental function evaluations" I believe there to be an errata in the hard coded coefficients for the 14th and 15th derivatives of the 'f' function copied directly from the original paper.

The derivative functions follow simple differentation. I believe an extraneous power has been added to the coefficients for the f14 and f15 variables - as a direct read across of an error in the original paper. 
Thus, 
src/poliastro/core/propagation/pimienta.py lines 142 and 143 should read:
```
    f14 = 1.307674368e12 * c15 * x_bar
    f15 = 1.307674368e12 * c15
```
not
```
    f14 = 1.307674368e13 * c15 * x_bar
    f15 = 1.307674368e13 * c15

```
Due to the nature of this propagation routine this is unlikely to have much impact on accuracy, but should be mathematically correct nonetheless.

**Comment by astrojuanlu** [comment]
Hello @amacki3 , thanks a lot for this investigation! I totally agree with you that the impact on accuracy is probably hard to detect, but we should definitely strive for correctness.

We are talking about this:

https://github.com/poliastro/poliastro/blob/1cc6b04efef353d52ed2f15991e58b3076b399a4/src/poliastro/core/propagation/pimienta.py#L142-L143

(used the "copy permalink" functionality)

Do you want to change the coefficients yourself? You can click the ✏️ "Edit" button on the file and the user interface will guide you towards sending a pull request.

**Comment by astrojuanlu** [comment]
Closed in #1473 !
